src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String summary;
+    protected boolean isComplete;
+
+    public Task(String summary) {
+        this.summary = summary;
+        this.isComplete = false;
+    }
+
+    public String getStatus() {
+        return (isComplete ? "[Y]" : "[N]");
+    }
+
+    public String getSummary() {
+        return summary;
+    }
+
+    public void markComplete() {
+        this.isComplete = true;
+    }
+
```

Comment:
Should the setter method for isComplete follow the form of `void setComplete(boolean isComplete);`?

Reply:
I would consider this differently from a setter method since this method does not take in an argument. But I agree there would be more flexibility if this method is converted to a setter method.